index.bs

```diff
@@ -1824,6 +1853,76 @@ object |transport|, and a |sendOrder|, run these steps.
 
 </div>
 
+# Interface `WebTransportRateControlFeedback` #  {#rate-control-feedback}
+
+A {{WebTransportRateControlFeedback}} contains information
+useful for the application to control the rate at which it sends data.
+
+<pre class="idl">
+[Exposed=(Window,Worker), SecureContext]
+interface WebTransportRateControlFeedback {
+  readonly attribute unsigned long long? sendRate;
+};
+</pre>
+
+## Internal slots ## {#rate-control-feedback-internal-slots}
+
+A {{WebTransportRateControlFeedback}} has the following internal slots.
+
+<table class="data" dfn-for="WebTransportRateControlFeedback" dfn-type="attribute">
+ <thead>
+  <tr>
+   <th>Internal Slot
+   <th>Description (<em>non-normative</em>)
+  </tr>
+ </thead>
+ <tbody>
+  <tr>
+   <td><dfn>`[[SendRate]]`</dfn>
+   <td class="non-normative">The rate at which the {{WebTransport}} dequeues and sends data,
+   which is the maximum rate the applcation should send at to avoid queuing.
+  </tr>
+ </tbody>
+</table>
+
+## Attributes ##  {#rate-control-feedback-attributes}
+
+: <dfn for="WebTransportRateControlFeedback" attribute>sendRate</dfn>
+:: The getter steps are to return [=this=]'s {{WebTransportRateControlFeedback/[[SendRate]]}}.
+   This is the rate at which queued data will be dequeued and sent by the user agent, in bits per second.
+   If the application wishes to avoid queuing, it can send at a rate less than this value.
+   This rate applies to all streams and datagrams that share a [=session=]
+   and is calculated using {{WebTransport/congestionControl}}. If this [=session=] is
+   pooled with others in a shared [=connection=], it is the rate at which the user
+   agent will drain the queues for this [=session=].  If the user agent cannot
+   determine a value (perhaps because of pooling), it may leave the value unset.
```

Comment:
Leaving the value unset seems unfortunate, and possibly bad for web compat. Should we encourage user agents to make up a value?